traveler-swift-core/TravelerKit/Models/Flight.swift

```diff
@@ -46,6 +52,8 @@ public struct Flight: Codable, Equatable {
         try container.encode(number, forKey: .number)
         try container.encode(departureAirport, forKey: .departureAirport)
         try container.encode(arrivalAirport, forKey: .arrivalAirport)
+        try container.encode(departureTerminal, forKey: .departureTerminal)
```

Comment:
Likewise we should ask the backend to put that in the airport model.

Reply:
Added TODO